js/csrfprotector.js

```diff
@@ -19,7 +19,10 @@ var CSRFP = {
 	 *
 	 * @var string array
 	 */
-	checkForUrls: [],
+	checkForUrls: Array.prototype.slice.call(document.getElementsByName("checkForUrls"))
```

Comment:
This line should work if there are no elements with name `checkForUrls`.
```lang=js
-> [].map(function (element) { return element.value; });
<- []

-> Array.prototype.slice.call(document.getElementsByName("badInputName"));
<- []
```
So, the worst thing that could happen is that `checkForUrls` is set to `[]`, which is the original behavior.

I do agree with you on changing the name, though.